tests/src/zigevm.ts

```
@@ -0,0 +1,186 @@
+    return this.wasmExports.zig_add(a, b);
+  }
+  
+  // Helper methods for memory management
+  
+  private hexToBytes(hex: string): Uint8Array {
+    // Remove 0x prefix if present
+    hex = hex.startsWith('0x') ? hex.slice(2) : hex;
+    
+    // Ensure even length
+    if (hex.length % 2 !== 0) {
+      hex = '0' + hex;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)